packages/core/src/scrim/Scrim.tsx

```diff
@@ -25,6 +23,7 @@ export const Scrim = forwardRef<HTMLDivElement, ScrimProps>(function Scrim(
   { className, children, fixed = false, open = true, ...rest },
   ref
 ) {
+  const withBaseName = makePrefixer("saltScrim");
```

Comment:
Had to move this to inside the Scrim component, otherwise returning 'cannot access makePrefixer before initialisation'

Reply:
All of our other components work without this being inside